packages/lesswrong/components/posts/DraftsList.tsx

```diff
@@ -0,0 +1,131 @@
+  terms: PostsViewTerms,
+  title?: string,
+  showAllDraftsLink?: boolean,
+  classes: ClassesType
+}) => {
+  const currentUser = useCurrentUser();
+  const { PostsItem2, Loading } = Components
+  
+  const { query } = useLocation();
+  const [showSettings, setShowSettings] = useState(false);
+  
+  const {mutate: updatePost} = useUpdate({
+    collectionName: "Posts",
+    fragmentName: 'PostsList',
+  });
+  
+  const toggleDelete = useCallback((post) => {
+    // if (post.deletedDraft||confirm("Are you sure you want to delete this post?")) { //don't confirm to undelete
+    void updatePost({
+      selector: {_id: post._id},
+      data: {deletedDraft:!post.deletedDraft, draft: true} //undeleting goes to draft
+    })
+    // }
+  }, [updatePost])
+  
+  const { results, loading, loadMoreProps } = useMulti({
+    terms,
+    collectionName: "Posts",
+    fragmentName: 'PostsList',
+    enableTotal: true,
+    fetchPolicy: 'cache-and-network',
+    nextFetchPolicy: "cache-first",
+  });
+  
+  if (!currentUser) return null
+  if (!results && loading) return <Loading />
+  
+  const currentSorting = query.sortDraftsBy || query.view ||  "lastModified"
+  const currentIncludeEvents = (query.includeDraftEvents === 'true')
+  const currentIncludeArchived = (query.includeArchived === 'true')
+  terms.excludeEvents = !currentIncludeEvents 
+  
+  
+  return <div>
+    <Components.SectionTitle title={title}>
+      <div className={classes.draftsHeaderRow}>
+        <div className={classes.newPostButton}>
+          <Link to={"/newPost"}>
+            <Components.SectionButton>
+              <DescriptionIcon /> New Post
+            </Components.SectionButton>
+          </Link>
+        </div>
+        {showAllDraftsLink && <div className={classes.draftsPageButton}>
+          <Link to={"/drafts"}>
+            <Components.SectionButton>
+              <ListIcon /> All Drafts
+            </Components.SectionButton>
+          </Link>
+        </div>}
+        <div className={classes.settingsButton} onClick={() => setShowSettings(!showSettings)}>
+          <Components.SettingsButton label={`Sorted by ${ sortings[currentSorting]}`}/>
+        </div>
+      </div>
+    </Components.SectionTitle>
+    {showSettings && <Components.DraftsListSettings
+      hidden={false}
+      currentSorting={currentSorting}
+      currentIncludeEvents={currentIncludeEvents}
+      currentIncludeArchived={currentIncludeArchived}
+      sortings={sortings}
+    />}
+    {results
+      .map((post: PostsList, i: number) =>
+      <PostsItem2
+        key={post._id} 
+        post={post}
+        draft
+        toggleDeleteDraft={toggleDelete}
+        hideAuthor
+        showDraftTag={false}
+        showPersonalIcon={false}
+        showBottomBorder={i < results.length-1}
+        strikethroughTitle={post.deletedDraft}
+      />
+    )}
+    <Components.LoadMore {...{...loadMoreProps, count: undefined, totalCount: undefined }}/>
```

Comment:
You could just not pass `enableTotal: true` to `useMulti`, that's simpler (and more performant) than editing the `count` and `totalCount` fields out of `loadMoreProps`.